decred/decred/dcr/nets.py

```diff
@@ -24,3 +24,16 @@ def parse(name):
         if net.Name == name:
             return net
     raise Exception("unrecognized network name %s" % name)
+
+
+def normalizeNetName(netName):
```

Comment:
Little stutter here. Module name is **nets**, so `normalizeName` is enough, since it's use will be `nets.normalizeName`.